services/skipchain/skipchain.go

```diff
@@ -456,6 +461,20 @@ func (s *Service) lenSkipBlocks() int {
 	return len(s.SkipBlocks)
 }
 
+// saves the actual identity
+func (s *Service) save() {
```

Comment:
The path method is for setting or retrieve the path or both ?
Definitely an issue to do later, so we can let these functions as is for the moment